.github/workflows/notebook-controller-images-updater.yaml

```diff
@@ -17,6 +17,10 @@ on:
         required: true
         default: "true"
   workflow_call:
+    secrets:
+      GH_TOKEN:
+        description: 'A token passed from the caller workflow'
+        required: true
```

Comment:
I see.

```
if [ -z "${{ env.GH_TOKEN }}" ]; then
```

But still, the GH_TOKEN is a required secret, `required: true`, so how does an user leave it empty when running manually, so that the if triggers

Reply:
Is a required secret when is called from another action `workflow_call` not by the user